runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -59,13 +59,13 @@
 
 	private static final String SEARCH_BANK_INPUT_TEXT =
 		"Show items whose names or tags contain the following text:<br>" +
-		"(To show only tagged items, start your search with 'tag:')";
+		"(To show only tagged items, start your search with 'tag:' or '#')";
 
 	private static final String SEARCH_BANK_INPUT_TEXT_FOUND =
 		"Show items whose names or tags contain the following text: (%d found)<br>" +
-			"(To show only tagged items, start your search with 'tag:')";
+			"(To show only tagged items, start your search with 'tag:' or '#')";
 
-	private static final String TAG_SEARCH = "tag:";
+	private static final String TAG_SEARCH = "^(?:tag:|#)(.+)";
```

Comment:
Typically using a pre-compiled `Pattern` is preferred to using a regex `String`. Can you change this to use a `Pattern` instead?

Reply:
How do you do string replacement with a `Pattern`? `String.matches()` and `String.replaceAll()` both expect regexes as strings

Reply:
You get the group, not string replace.

Reply:
done